.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and deploy # Note: Deployment will be added in a future PR 
+
+on:
+  push:
+    branches-ignore:
+      - develop
+      - master
```

Comment:
how come we are ignoring master for this build and deploy workflow?

Reply:
This is based on our circle CI config where master has a separate deploy process (the build_and_hold) which requires manual approval. That config is scheduled for future items

Reply:
ah gotcha